tests/cli/test_bashlib.py

```diff
@@ -104,10 +104,8 @@ def test_bashlib_minversion(self):
         exit_code, out, err = self.invoke_bash(
             "source $(ocrd bashlib filename) && ocrd__minversion 2.29.0")
         assert exit_code == 0
-        (major, minor, patch) = map(int, str(VERSION).split('.'))
```

Comment:
IIRC I did introduce this parsing to make that it's actually three tokens which are valid `int` because `ocrd__minversion` requires that. We need to fix it in `lib.bash`, otherwise:

```bash
> source core/src/ocrd/lib.bash
> (ocrd__minversion 3.0.0a1)
bash: ((: 0a1: value too great for base (error token is "0a1")
ERROR: ocrd/core is too old (3.0.0a1 < 3.0.0a1). Please update OCR-D/core
```

Reply:
https://github.com/bertsky/core/pull/7

Reply:
Oh, sorry!